docs/design/data-loading.mdx

```diff
@@ -978,8 +978,11 @@ SNOWFLAKE_DEFAULT_SCHEMA: null              # Optional default schema
 SNOWFLAKE_PRIVATE_KEY_PASSPHRASE: null      # Optional private key passphrase
 SNOWFLAKE_PRIVATE_KEY_PATH: null            # Optional private key path
 SNOWFLAKE_ROLE: null                        # Optional role name
+SNOWFLAKE_TABLE_NAME: null                  # Optional table name
```

Comment:
We don't need this config field

Reply:
Removed on c92a9a19635f3ec527768bebdfa3cc8a4d55613f